packages/mgt/src/graph/graph.user.ts

```diff
@@ -420,3 +420,65 @@ export async function findUsers(graph: IGraph, query: string, top: number = 10):
   }
   return graphResult ? graphResult.value : null;
 }
+
+/**
+ * async promise, returns all matching Graph users who are member of the specified group
+ *
+ * @param {string} query
+ * @param {string} groupId - the group to query
+ * @param {number} [top=10] - number of people to return
+ * @param {PersonType} [personType=PersonType.person] - the type of person to search for
+ * @param {boolean} [transitive=false] - whether the return should contain a flat list of all nested members
+ * @returns {(Promise<User[]>)}
+ */
+export async function findUsersFromGroup(
+  graph: IGraph,
+  query: string,
+  groupId: string,
+  top: number = 10,
+  personType: PersonType = PersonType.person,
+  transitive: boolean = false
+): Promise<User[]> {
+  const scopes = ['user.read.all', 'people.read'];
+  const item = { maxResults: top, results: null };
+
+  let cache: CacheStore<CacheUserQuery>;
+  const key = `${groupId || '*'}:${query || '*'}:${personType}:${transitive}`;
+
+  if (usersCacheEnabled()) {
+    cache = CacheService.getCache<CacheUserQuery>(cacheSchema, queryStore);
+    const result: CacheUserQuery = await cache.getValue(key);
+
+    if (result && getUserInvalidationTime() > Date.now() - result.timeCached) {
+      return result.results.map(userStr => JSON.parse(userStr));
+    }
+  }
+
+  let filter: string = '';
+  if (query) {
+    filter = `startswith(displayName,'${query}') or startswith(givenName,'${query}') or startswith(surname,'${query}') or startswith(mail,'${query}') or startswith(userPrincipalName,'${query}')`;
+  }
+
+  let apiUrl: string = `/groups/${groupId}/${transitive ? 'transitiveMembers' : 'members'}`;
+  if (personType === PersonType.person) {
+    apiUrl += `/microsoft.graph.user`;
+  } else if (personType === PersonType.group) {
+    apiUrl += `/microsoft.graph.group`;
+  }
```

Comment:
Now, should this be done or should we force the return type by filtering on the type of objects we want? Here, if `any` is passed in, it will return both users AND groups. Potentially creating duplicated down the line (we are filtering them out... But we get more data than we should). @nmetulev your input will be very valuable here! Thanks!